hapic/processor/main.py

```diff
@@ -213,3 +214,10 @@ def _get_ouput_file_validation_error_message(
         elif data.file_object and not data.mimetype:
             error_message = "File object should have explicit mimetype"
         return error_message
+
+    @classmethod
+    @abc.abstractmethod
+    def get_default_error_builder(cls) -> "ErrorBuilderInterface":
```

Comment:
semantically,  the order of decorators should be in the other order:

We define : 

1. a get_default_error_builder method
2. this method is a class method
3. here, the method is abstract because this is only the interface.

Is it working if you put the decorators in this order?

Reply:
i have a warning in opposite order of decorators, i don't know know why:

![selection_026](https://user-images.githubusercontent.com/1104637/50292097-8d00d100-0470-11e9-8adf-19c04fe9a71d.png)

Reply:
ok. so add a comment like: "the decorators order is not semantically right, but in right order pycharm generates a warning: xxx"